manifest.json

```diff
@@ -28,25 +28,23 @@
 			{
 				"name": "domain",
 				"type": "domain",
-                		"ask": {
-   					"en": "Choose a domain for Mopidy",
-                    			"fr": "Choisissez un domaine pour Mopidy"
-                			},
+        "ask": {
+          "en": "Choose a domain for Mopidy",
+          "fr": "Choisissez un domaine pour Mopidy"
+                },
```

Comment:
```suggestion

```

That's no more needed :)

Reply:
Thanks for reviewing
I deleted the ask-attributes. It works on commandline (yunohost install http...) but not with webUI, where I waited 15 minutes looking at the "pacman eater" .

Reply:
Same effect with the webui ://// I don't know why, maybe we should ask @alexAubin why if we remove the ask question for domain in manifest.json... we have and endless pacman...

Reply:
Hmpf yeah, turns out that break the webadmin "custom app install" because the webadmin downloads the raw manifest.json from the repo and expects to find the "ask" key in each question ...

For this reason and [other reasons](https://github.com/YunoHost/yunohost-admin/pull/207#issuecomment-803370120), we should rework the way the custom app install work, such that downloading the manifest.json would be done by the backend, which would also inject the default ask string (as it's done for regular installations via the catalog) to make everything consistent...

Reply:
So I 'll stick to the ask string until the final solution is found